lib/datasource/gitlab-tags/index.ts

```diff
@@ -19,9 +19,11 @@ function getCacheKey(depHost: string, repo: string): string {
 }
 
 export async function getReleases({
-  registryUrl: depHost,
+  registryUrl,
   lookupName: repo,
 }: GetReleasesConfig): Promise<ReleaseResult | null> {
+  const depHost = registryUrl.replace(/\/api\/v4$/, '');
```

Comment:
What was your reasoning for stripping the `/api/v4` but then adding it back later? Don't all requests need `/api/v4` as their prefix?

Reply:
`/api/v4` shoulod not be contained in `registryUrl`. We use the url later for generating the user repo url, with previously was `https://domain.com/api/v4/some/repo` but it should be `https://domain.com/some/repo` for user facing urls